components/services/bat_ledger/bat_ledger_client_mojo_proxy.cc

```diff
@@ -695,6 +708,30 @@ void BatLedgerClientMojoProxy::GetActivityInfoList(uint32_t start,
       base::BindOnce(&OnGetActivityInfoList, std::move(callback)));
 }
 
+void OnURLRequest(const confirmations::URLRequestCallback& callback,
+                  int32_t status_code,
+                  const std::string& content,
+                  const base::flat_map<std::string, std::string>& headers) {
+  callback(status_code, content, ToStdMap(headers));
+}
+
+void BatLedgerClientMojoProxy::URLRequest(
+    const std::string& url,
+    const std::vector<std::string>& headers,
+    const std::string& content,
+    const std::string& content_type,
+    ledger::URL_METHOD method,
+    ledger::URLRequestCallback callback) {
+  if (!Connected()) {
+    callback(418, "", std::map<std::string, std::string>());
```

Comment:
This HTTP response seems random (I copied it from similar functionality in bat_ledger). Should it just be a generic 400 (and preferably some kind of constant - Chromium probably has definitions for these).

Reply:
This was chosen as an unlikely response that we would get in the real world so that we knew the service had failed. @bridiver your thoughts please?

Reply:
this shouldn't be here at all. The mojo proxy shouldn't have any logic in it